themes/clean-landing/layouts/partials/implementations.html

```diff
@@ -3,85 +3,27 @@
   <div class="content-wrapper">
     
     <div class="subtitle has-text-centered">
-      <h4>Implementations</h4>
+      <h4>Ecosystem</h4>
     </div>
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
+      <div class="column is-one-fifths is-one-fifths-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
-      <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
-        <small><a href="https://crossplane.io">crossplane.io</a></small>
+      <div class="column is-four-fifths is-four-fifths-mobile">
+        <p><b>Crossplane</b>: Crossplane implements OAM to deliver modern applications with cloud resources in a standard approach.</p>
```

Comment:
Just copy from original section:

Crossplane is the open source Kubernetes add-on that enables you to provision and manage OAM applications and the infrastructure they run on directly from Kubernetes.

others LGTM, thanks!